.circleci/config.yml

```diff
@@ -427,7 +427,7 @@ jobs:
     resource_class: xlarge
     docker:
       # Need to use docker image for these steps.
-      - image: circleci/python:2.7.14
+      - image: circleci/python:3.10
```

Comment:
why are we using python image here? Mayflower is using `image: cimg/node:12.14`. Should we make them consistent?

Reply:
Although, the updated python image seems to work in circleCI for openmass, see report https://app.circleci.com/pipelines/github/massgov/openmass/19062/workflows/4af91a45-756e-4730-a2ee-76d268c19df3/jobs/79123/artifacts, when I pull down the image and run patternlab backstop locally its having issues displaying the images. 

<img width="1751" alt="Screen Shot 2022-05-09 at 8 40 23 PM" src="https://user-images.githubusercontent.com/5789411/167644623-d6947393-339c-4aa8-bd9c-977549f08922.png">

Reply:
nvm, this could be totally unrelated...